tests/rptest/services/redpanda_cloud.py

```diff
@@ -657,7 +657,7 @@ def _create_new_cluster(self):
                                f"'{self.config.region}'")
 
         # Call Api to create cluster
-        self._logger.info(f'creating cluster name {self.current.name}')
+        self._logger.warning(f'creating cluster name {self.current.name}')
```

Comment:
What's the significance of log level bump?

Reply:
Basically within the test code itself nothing at INFO or below gets emitted to the stdout, only to the log files on disk. WARN+ goes to stdout too, so it is displayed in the buildkite live terminal log. Until the run finishes, only the terminal log is avaialble and in many failure modes the "other" logs are not available even after the run finishes.

Since cluster ID is among the most key pieces of info for a cloud run we put it at WARN so it is always available immediately and not lost in the other failure modes.